packages/relay/src/synchornizer.ts

```diff
@@ -59,30 +59,124 @@ export class UnirepSocialSynchronizer extends Synchronizer {
         const postId = BigInt(event.topics[2]).toString()
         const epoch = Number(event.topics[3])
         const content = decodedData.content
 
-        db.upsert('Post', {
-            where: {
-                _id: findPost._id,
-            },
-            create: {
+        console.log('----handlePost----')
+        console.log('Epoch key:', epochKey)
+        console.log('Post id:', postId)
+        console.log('Epoch:', epoch)
+        console.log('Content:', content)
+        console.log('--------------------')
+
+        if (findPost) {
+            db.update('Post', {
+                where: {
+                    _id: findPost._id,
+                },
+                update: {
+                    status: 1,
+                    postId,
+                },
+            })
+        } else {
+            db.create('Post', {
                 postId,
                 epochKey,
                 epoch,
                 transactionHash,
                 status: 1,
                 content,
+                upCount: 0,
+                downCount: 0,
+                commentCount: 0,
+            })
+        }
+
+        return true
+    }
+
+    async handleComment({ event, db, decodedData }: EventHandlerArgs) {
+        const transactionHash = event.transactionHash
+        const epochKey = BigInt(event.topics[1]).toString(10)
+        const postId = BigInt(event.topics[2]).toString()
+        const commentId = BigInt(event.topics[3]).toString()
+        const epoch = Number(decodedData.epoch)
+        const content = decodedData.content
+
+        console.log('----handleComment---')
+        console.log('Epoch key:', epochKey)
+        console.log('Post id:', postId)
+        console.log('Comment id:', commentId)
+        console.log('Epoch:', epoch)
+        console.log('Content:', content)
+        console.log('--------------------')
+
+        const findComment = await this.db.findOne('Comment', {
+            where: {
+                transactionHash,
             },
-            update: {
+        })
+        if (findComment) {
+            db.update('Comment', {
+                where: {
+                    _id: findComment._id,
+                },
+                update: {
+                    commentId,
+                    status: 1,
+                },
+            })
+        } else {
+            db.create('Comment', {
+                commentId,
+                postId,
+                transactionHash,
+                content,
+                epoch,
+                epochKey,
                 status: 1,
+            })
+        }
+
+        // atrieve the comment count of the post
+        const commentCount = await this.db.count('Comment', {
+            postId,
+        })
+
+        db.update('Post', {
+            where: {
                 postId,
             },
+            update: {
+                commentCount: commentCount + (findComment ? 0 : 1),
```

Comment:
After create or update the comment, not sure this count will become correct one? 
`commentCount: commentCount`